axon-server-connector/src/test/java/org/axonframework/axonserver/connector/query/StreamingQueryEndToEndTest.java

```diff
@@ -167,6 +180,26 @@ void testStreamingFluxQuery(boolean supportsStreaming) {
                     .verifyComplete();
     }
 
+    @ParameterizedTest
+    @ValueSource(booleans = {true, false})
+    void testConcurrentStreamingQueries(boolean supportsStreaming) throws InterruptedException, ExecutionException {
+        StreamingQueryMessage<FluxQuery, String> query =
+                new GenericStreamingQueryMessage<>(new FluxQuery(), String.class);
+
+        AtomicReference<Throwable> error = new AtomicReference<>(null);
+        CountDownLatch countDownLatch = new CountDownLatch(100);
+        for (int i = 0; i < 100; i++) {
+            streamingQueryPayloads(query, supportsStreaming)
```

Comment:
Why don't we use a StepVerifier for this?

Reply:
We want to have as many as possible parallel queries running, causing the executor to put them in the queue, verifying the queue will accept them. `StepVerifier` would sequentialize them :)

Reply:
Because we thought it didn't work. Apparently that was for another reason, because not it does an fails appropriately. Changing this.